longevity_test.py

```diff
@@ -51,6 +51,8 @@ def _run_all_stress_cmds(self, stress_queue, params):
                 stress_queue.append(self.run_stress_thread(stress_cmd=stress_cmd,
                                                            stats_aggregate_cmds=False,
                                                            round_robin=self.params.get('round_robin')))
+            elif stress_cmd.startswith('cassandra-harry'):
+                stress_queue.append(self.run_stress_thread(**params))
```

Comment:
why it is needed ?

Reply:
@amoskong it's exact same line as in the "else"